signbank/dictionary/templates/dictionary/gloss_detail_advanced.html

```diff
@@ -75,24 +75,14 @@ <h2 id='modalTitle'>{% blocktrans %}Delete This Morphology{% endblocktrans %}</h
             </table>
 
             {% if perms.dictionary.change_gloss %}
-            <form id='add_morphologydefinition_form' method='post' action='{% url 'dictionary:add_morphologydefinition' %}'>
```

Comment:
Same comment here - basically it would be good to keep the morphology table for now if we can, because I'm not sure if we'll be using it or not.